.circleci/config.yml

```diff
@@ -38,6 +38,9 @@ jobs:
           name: libs repo init
           command: |
             cd libs
+            rm -rf node_modules/web3-providers-ws/node_modules/websocket/.git/
+            source ~/.bash_profile
+            nvm install 10.16
```

Comment:
these lines are needed to get CI libs install step to work

Reply:
what node version already exists here?